blackfootwords/datatables.py

```diff
@@ -1,9 +1,24 @@
 from sqlalchemy.orm import joinedload
 from clld.web import datatables
 from clld.web.datatables.base import LinkCol, Col, LinkToMapCol
+from clld.web.datatables.value import Values
+from clld.db.util import get_distinct_values
 
 from blackfootwords import models
 
 
+class Lemmas(Values):
+    def col_defs(self):
+        res = Values.col_defs(self)
+        res.append(Col(
+            self,
+            'categories',
+            model_col=models.Lemma.categories,
+            choices=get_distinct_values(models.Lemma.categories),
+        ))
```

Comment:
... then add our new column. (Informing the app about the corresponding column in the database, to make searching and sorting work; and since it's a column with a somewhat fixed set of values, we turn the search box into a dropdown list by specifying the available "choices".) ...